.gitignore

```diff
@@ -1 +1,8 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
+.ipynb_checkpoints
+*.ipynb
+venv
+env
```

Comment:
I think this should also be on the user level, because user decides how to call them